adafruit_fram.py

```diff
@@ -146,8 +146,8 @@ def __getitem__(self, key):
             registers = list(range(key.start if key.start is not None else 0,
                                    key.stop if key.stop is not None else self._max_size))
             if (registers[0] + len(registers)) > self._max_size:
-                raise ValueError("Register + Length greater than maximum FRAM size."
-                                 " ({0})".format(self._max_size))
+                raise ValueError("Register + Length greater than maximum FRAM register."
```

Comment:
On this one, what do you think some good wording would be for the error message since it's a a slice of multiple addresses? I was thinking something along the lines of:
`Address slice out of range. It must be 0 <= address + length < {0}.`